docs/configuration.rst

```diff
@@ -325,6 +349,50 @@ Removes a variant set from a given data repository and dataset.
 
     $ ga4gh_repo remove-variantset path/to/datarepo aDataset aVariantSet
 
++++++++++++++++
+add-biosample
++++++++++++++++
+
+Add a BioSample record to the given data repository and dataset. Records
+should be described in JSON format according to the GA4GH protocol. To 
+associate this BioSample with an Individual record add the optional 
+individualName flag.
```

Comment:
But:
* Where is the uniqueness required? (Sorry; I just may have lost track.)
* ID is used for references, never name.
* Even though I understand that "name" in the schema may not mean "Walter White", it hurts a bit to have "name" of "individual"  being a unique reference point.